source/renderer/app/api/assets/types.js

```diff
@@ -1,77 +1,88 @@
 // @flow
 import BigNumber from 'bignumber.js';
 
-import Asset from '../../domains/Asset';
+import AssetDomain from '../../domains/Asset';
 
-export type SingleAsset = {
+/**
+ *
+ * ASSET
+ * Fetched from the Assets API endpoint
+ * It's not attached to a particular wallet or transaction
+ * Therefore, it doesn't have `quantity` nor `address`
+ *
+ * Exclusive data: fingerprint, metadata
+ * Missing data: quantity, address
+ *
+ */
+export type ApiAssets = Array<ApiAsset>;
+export type ApiAsset = {
   policy_id: string,
   asset_name: string,
   fingerprint: string,
   metadata?: ?AssetMetadata,
 };
-
-export type ApiAsset = {
-  policy_id: string,
-  asset_name: string,
+export type Asset = {
+  assetName: string,
+  decimals: ?number,
   fingerprint: string,
   metadata?: ?AssetMetadata,
+  policyId: string,
+  recommendedDecimals: ?number,
+  uniqueId: string,
 };
 
-export type AssetMetadata = {
-  name: string,
-  ticker: string,
-  description: string,
-  unit?: {
-    decimals: number,
-    name: string,
-  },
-  url?: string,
-  logo?: string,
-};
-
-export type AssetItem = {
+/**
+ *
+ * TOKEN
+ * Asset that is attached to a particular wallet and/or transaction
+ * It doesn't have the Asset details (fingerprint, metadata)
+ *
+ * Exclusive data: quantity, address
+ * Missing data: fingerprint, metadata
+ *
+ */
+export type ApiToken = {
   policy_id: string,
   asset_name: string,
   quantity: number,
+  address?: ?string,
 };
-
-export type WalletAssetItem = {
+export type ApiTokens = Array<ApiToken>;
```

Comment:
same here

Reply:
In this case it's already after `ApiToken`, just before `Token`